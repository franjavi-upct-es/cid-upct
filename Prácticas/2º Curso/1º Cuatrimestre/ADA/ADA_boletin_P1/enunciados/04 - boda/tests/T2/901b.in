53
74 15
3 1 9 13
6 10 11 5 10 20 7
6 16 22 22 20 15 20
5 11 28 20 1 7
13 10 5 21 15 30 27 15 22 5 19 9 6 21
7 15 10 11 6 18 3 27
13 17 9 27 19 28 17 19 5 25 28 9 15 13
4 11 19 21 8
4 21 13 20 26
2 22 28
5 9 30 22 7 8
14 3 26 19 11 6 23 6 26 24 12 8 23 23 18
6 30 17 18 12 7 5
11 28 3 4 28 24 25 4 2 16 28 19
6 9 25 19 6 20 12
46 19
8 1 6 23 22 23 11 26 21
13 26 10 17 29 7 11 15 10 4 22 7 22 26
13 8 6 12 19 17 13 7 12 13 5 5 27 19
11 22 9 13 9 18 30 30 16 2 14 25
3 6 1 10
7 8 10 20 11 28 6 15
7 9 20 1 5 16 11 11
1 19
10 7 28 15 28 5 8 12 29 26 9
3 5 23 20
6 12 23 4 9 7 30
13 26 22 14 11 2 16 1 12 1 30 1 15 27
7 14 30 25 1 8 15 28
8 5 3 25 19 6 3 25 27
4 21 10 25 23
1 10
14 6 11 15 6 17 11 2 1 3 26 1 10 3 28
6 29 1 18 17 28 21
1 17
98 13
14 26 8 29 27 23 4 7 7 2 24 9 25 16 11
9 16 13 15 6 28 5 28 15 21
13 27 23 3 20 8 21 7 15 11 4 29 14 10
11 7 3 27 2 18 8 14 26 12 20 23
11 24 12 15 6 20 4 20 15 23 27 5
4 3 15 25 1
4 4 19 27 29
8 20 8 15 3 25 26 14 9
12 29 20 2 4 2 5 23 16 28 12 20 19
8 26 5 7 15 30 17 3 20
12 22 28 16 16 14 3 29 23 20 27 4 22
10 5 26 7 12 15 18 1 25 23 26
7 21 3 29 8 5 18 1
64 17
7 4 21 11 25 13 1 13
14 22 5 13 9 3 16 24 20 17 18 4 4 9 17
10 7 24 3 16 24 21 23 2 16 5
5 10 9 5 23 16
8 27 20 26 21 6 11 10 14
13 5 17 29 21 7 27 6 9 12 22 21 26 15
12 30 19 16 8 23 8 16 10 26 5 28 16
12 8 17 23 28 13 2 26 26 8 22 1 8
2 14 20
13 29 25 13 17 2 12 2 1 27 11 26 24 8
11 25 16 11 10 5 16 3 1 11 10 14
9 9 1 17 28 14 15 14 18 23
4 21 24 16 10
7 4 25 4 28 19 11 1
8 16 16 30 8 26 1 13 28
3 6 6 28
7 12 4 20 5 19 10 20
74 11
2 29 5
2 27 24
10 27 13 21 4 5 20 29 5 3 18
8 30 16 17 10 27 20 29 23
8 30 13 26 3 20 24 7 1
10 22 15 30 5 5 3 1 25 23 27
7 3 24 18 18 11 19 14
6 9 29 22 8 11 17
14 30 10 9 23 14 30 29 13 26 3 16 26 19 30
11 29 2 16 8 11 18 18 17 10 26 15
11 26 17 17 27 8 18 5 30 1 27 28
99 12
11 21 10 3 20 2 24 14 9 1 24 19
7 10 28 14 24 20 1 10
10 28 18 15 2 9 16 28 7 29 12
1 11
7 15 30 14 8 5 14 9
3 2 19 8
5 24 24 10 25 3
13 22 12 22 15 21 29 5 19 19 16 1 29 6
13 20 11 23 25 24 23 23 18 3 22 30 26 15
4 12 10 8 25
9 21 10 3 11 14 13 29 21 13
1 18
81 17
10 4 3 13 18 17 30 20 8 30 7
13 1 11 24 30 5 6 13 6 1 23 11 13 22
11 18 18 10 29 26 7 2 20 20 19 6
11 30 14 10 28 28 2 8 13 2 5 18
14 2 10 6 5 23 19 27 2 7 7 30 24 5 1
4 16 11 19 27
4 24 28 29 13
6 29 17 23 25 27 28
6 6 25 22 28 14 19
5 12 17 20 27 21
2 2 7
2 13 25
3 6 22 14
1 13
14 27 27 28 23 24 15 29 11 7 18 16 17 9 27
11 28 15 15 2 17 13 23 29 7 30 26
9 5 28 3 8 25 30 5 9 23
84 19
5 17 17 10 25 17
4 27 6 13 4
7 29 16 22 19 23 21 6
13 25 4 15 24 20 6 20 28 21 30 19 15 9
7 25 3 13 22 22 10 26
9 17 16 2 30 4 16 12 10 20
4 5 27 21 24
12 10 13 22 9 1 7 17 19 23 12 1 6
11 3 2 19 11 17 20 10 13 28 21 14
5 18 18 13 8 3
9 17 15 28 17 8 26 26 26 18
3 19 16 1
9 17 19 23 25 30 2 29 19 14
6 6 23 21 18 30 23
8 8 30 22 16 29 17 11 24
3 9 4 12
11 16 20 19 8 6 11 1 5 29 14 8
2 28 29
2 19 21
72 6
14 12 12 10 21 14 26 17 15 29 20 23 7 1 4
14 28 6 7 2 26 12 10 22 10 30 27 28 20 17
4 24 29 27 3
5 2 28 19 16 19
10 1 25 30 26 30 27 1 6 21 18
2 22 9
100 1
1 16
79 4
11 25 4 19 28 14 21 17 2 28 5 1
8 21 30 15 21 18 7 18 8
3 5 29 3
7 19 30 9 21 13 9 15
82 18
2 21 10
5 22 7 17 22 27
10 13 3 20 22 10 7 22 4 12 12
12 26 1 27 4 13 1 5 19 8 2 15 20
1 5
7 1 21 24 27 20 28 30
1 20
9 8 11 4 19 22 1 6 14 20
14 18 12 14 29 12 7 13 1 1 17 12 1 29 27
4 11 25 11 11
14 11 18 8 14 6 21 6 12 5 17 13 14 29 18
6 2 16 24 2 16 2
13 8 23 24 27 3 18 7 13 15 17 1 22 22
2 12 28
3 8 6 5
2 4 22
7 27 29 19 21 6 12 25
4 4 18 10 6
61 7
13 3 25 3 24 17 30 28 6 23 5 11 27 26
13 11 21 3 1 1 23 7 12 10 20 16 19 21
9 8 29 15 10 24 17 26 2 9
5 7 1 27 9 19
7 7 21 27 10 22 27 24
6 30 25 9 7 14 21
10 13 20 26 22 5 13 17 6 21 9
41 13
6 12 1 12 19 22 30
8 5 18 5 24 17 30 2 24
3 14 5 17
3 30 30 7
6 17 12 24 17 7 6
2 19 29
11 29 20 16 18 16 3 14 9 12 13 2
11 17 15 1 25 18 22 25 16 18 11 28
13 19 4 9 3 14 7 11 4 18 27 21 3 21
7 3 2 9 4 28 18 18
10 12 27 12 6 13 21 8 2 23 27
11 1 21 11 29 1 14 8 27 27 2 10
11 4 11 1 29 9 18 17 28 22 13 1
45 5
1 5
6 14 1 23 6 21 25
9 21 8 26 10 26 28 11 18 23
9 11 22 29 20 8 19 11 12 11
8 29 2 4 24 7 4 8 4
77 12
4 28 2 18 29
1 15
13 15 29 30 17 20 20 6 19 30 9 23 11 16
7 4 11 14 10 14 14 5
5 15 27 10 16 14
13 13 28 17 19 18 8 27 30 20 2 10 19 10
6 21 17 22 24 27 28
9 2 3 30 14 17 18 24 3 1
8 7 20 10 17 7 10 13 6
5 6 8 9 16 9
8 24 23 15 13 12 2 14 14
11 28 30 10 13 24 10 14 22 21 15 8
71 16
14 17 6 18 24 15 25 2 28 18 16 4 30 27 5
6 2 19 25 24 28 7
9 29 12 1 11 18 30 30 4 12
10 9 29 9 15 23 2 12 2 18 16
6 6 12 21 8 1 15
7 20 13 9 19 16 9 29
5 9 28 28 12 13
3 10 13 13
8 15 17 17 2 24 10 7 5
12 6 27 6 6 17 10 15 27 17 15 25 12
5 15 9 26 19 7
7 24 12 20 8 28 29 9
5 30 7 17 22 13
4 19 18 30 29
2 26 15
3 12 18 23
71 7
5 29 25 7 22 6
9 29 25 17 29 15 16 5 24 7
1 7
4 19 28 16 4
3 22 12 26
6 27 14 6 7 12 22
9 3 28 1 23 22 17 21 29 25
84 12
7 26 20 18 6 17 25 9
3 17 13 26
12 9 9 23 7 12 14 19 14 3 12 6 17
14 18 7 14 5 20 7 22 9 16 27 17 11 27 17
11 9 12 14 9 13 6 15 24 11 26 30
10 29 5 22 18 15 20 2 19 9 30
2 9 15
12 25 25 16 3 13 25 14 26 25 26 23 10
10 26 27 11 1 25 7 14 12 21 3
1 1
3 4 2 11
3 21 27 5
52 19
14 23 12 5 17 30 20 18 11 15 14 21 7 30 19
14 12 10 14 16 10 16 12 12 26 22 2 22 26 30
2 5 22
7 10 1 22 29 18 24 5
4 6 11 23 25
2 26 4
11 12 5 20 15 16 15 6 10 28 1 1
3 6 15 1
7 15 15 27 24 8 23 17
2 3 10
12 24 5 17 28 8 21 9 22 29 23 19 30
2 12 30
2 9 14
8 15 20 12 3 14 12 25 30
13 19 1 8 4 6 16 23 5 6 1 19 4 23
13 25 4 10 25 23 18 8 20 24 20 24 26 3
13 12 2 13 30 25 12 3 30 27 25 26 24 25
14 20 9 13 14 4 14 8 26 2 8 8 25 27 1
13 21 27 24 22 1 24 16 12 18 7 30 13 25
96 19
12 4 8 4 18 11 18 17 29 19 16 6 5
12 6 25 24 24 19 8 24 4 15 27 21 14
8 25 8 10 24 7 14 23 3
1 25
8 1 23 30 17 20 4 12 17
7 6 10 1 13 25 4 27
9 16 10 16 11 9 18 26 16 23
10 10 7 26 29 7 19 20 15 30 23
1 17
1 2
12 5 6 13 30 2 3 16 4 11 18 12 20
9 19 12 14 20 18 9 10 16 19
5 1 11 22 27 27
14 20 6 23 25 18 22 18 13 29 13 23 8 17 12
12 27 15 17 17 2 18 26 9 28 25 9 8
14 27 26 25 8 2 9 2 11 23 12 23 21 16 7
5 2 18 2 21 24
11 7 17 28 24 26 17 10 26 17 17 23
2 4 30
77 4
8 16 26 27 8 17 12 7 7
14 16 30 26 10 10 24 26 29 17 21 8 27 17 24
5 1 5 30 22 2
2 7 17
96 3
14 7 7 22 5 12 8 5 29 17 6 14 4 27 30
5 4 18 3 27 15
3 23 6 16
65 19
9 2 30 17 18 28 23 1 3 26
14 29 16 16 26 29 20 22 29 6 17 8 30 13 14
8 28 20 1 21 28 23 23 19
14 2 17 24 2 19 12 2 9 27 9 4 26 28 26
4 25 12 15 16
7 29 1 14 18 2 26 7
12 10 26 17 3 4 3 27 14 14 28 22 2
14 17 27 26 12 4 12 16 11 28 24 9 28 29 18
9 16 24 7 26 11 15 20 6 17
8 19 22 13 2 24 11 19 12
2 22 8
3 7 18 28
8 26 18 20 13 8 6 28 6
11 1 21 12 6 29 28 25 21 2 26 6
8 6 17 1 28 24 2 26 11
11 18 28 8 7 2 16 12 30 13 26 22
3 8 27 24
11 21 6 20 9 11 24 6 27 16 3 13
3 29 15 8
51 15
11 14 6 22 18 27 5 13 18 29 12 15
10 30 27 19 20 27 29 5 3 18 20
13 22 29 25 6 6 2 2 12 8 8 4 25 26
6 29 14 20 3 20 4
12 8 22 21 5 13 17 7 22 7 3 13 27
2 10 24
7 12 5 28 11 30 14 6
9 5 11 11 29 30 14 30 8 27
5 4 9 29 2 22
5 4 4 23 23 6
12 6 9 13 25 19 12 1 16 3 5 27 13
5 18 26 24 17 14
8 20 15 27 21 6 23 17 2
6 1 7 1 6 15 13
13 25 17 1 10 19 27 6 23 21 24 10 14 10
93 2
14 7 8 4 5 30 20 6 7 21 12 8 18 18 12
2 12 20
55 3
4 28 19 22 10
6 1 16 5 24 27 26
10 26 30 26 25 11 1 2 23 4 1
90 3
2 20 16
14 22 29 23 11 17 15 21 12 7 6 16 22 24 4
2 19 25
47 15
13 24 28 28 27 20 30 2 24 19 17 9 2 15
13 13 23 29 3 4 6 30 12 27 15 7 2 3
5 3 29 27 26 27
12 14 8 15 15 23 26 24 23 27 8 16 9
3 6 3 18
3 24 21 30
14 27 1 2 19 3 1 16 20 19 23 4 18 30 10
1 25
7 3 13 3 10 14 17 16
10 4 26 2 25 17 9 21 18 11 10
14 3 17 2 13 9 27 30 30 28 2 24 23 27 29
3 6 12 3
5 27 6 9 28 22
1 29
1 4
98 6
14 25 22 9 7 22 5 28 22 24 22 7 16 18 5
12 23 16 4 6 5 2 6 24 15 30 22 19
4 14 24 11 8
5 19 6 29 15 26
4 30 17 18 16
12 15 9 10 30 5 15 26 28 20 20 12 12
90 11
13 17 16 9 24 23 19 22 13 25 17 24 24 25
5 1 20 9 2 29
11 6 6 26 3 25 7 14 28 10 6 26
14 22 26 11 14 14 2 18 30 10 3 24 4 27 16
11 6 9 14 6 14 19 23 8 5 30 14
7 9 19 20 26 2 16 6
9 21 30 24 21 1 26 6 5 23
5 19 28 22 2 25
9 12 17 5 17 16 10 11 24 20
4 11 22 29 9
8 20 8 22 2 30 17 7 4
78 19
9 20 2 8 14 29 20 1 25 28
8 4 8 24 23 21 4 14 20
4 4 1 11 25
2 11 3
6 14 4 18 20 23 20
12 29 10 17 21 4 14 28 7 13 13 21 3
3 27 14 28
2 14 1
4 15 11 18 14
6 13 24 6 6 5 3
4 14 11 16 17
8 5 15 28 18 27 22 4 23
14 23 22 19 23 29 26 3 17 9 11 29 2 16 26
10 10 21 19 20 6 27 5 11 3 2
2 22 23
1 14
10 15 28 9 30 26 4 2 4 4 12
3 6 19 20
11 28 11 14 9 8 10 13 10 5 6 29
62 1
11 1 10 28 28 10 27 16 13 20 19 9
81 13
2 4 3
10 24 5 21 2 12 23 7 14 27 12
2 14 2
13 14 4 6 4 13 24 19 18 13 29 26 28 12
9 24 6 8 17 10 29 18 13 21
2 26 9
10 22 22 29 8 27 24 13 30 7 6
8 16 10 9 11 30 12 3 23
7 2 9 26 22 18 8 4
14 26 12 9 17 25 29 16 22 15 28 21 21 25 1
10 26 1 8 17 12 2 1 20 3 9
9 25 19 14 28 30 9 2 30 17
4 20 2 17 4
92 20
14 7 22 13 25 14 20 11 17 13 12 6 16 12 20
2 30 25
7 21 26 22 12 12 18 2
12 26 5 18 17 21 25 30 3 11 13 15 13
13 27 24 4 4 28 15 5 19 10 26 2 5 17
3 8 26 14
11 14 11 23 30 23 17 30 25 19 4 1
11 25 20 17 28 23 14 5 28 24 14 15
1 10
11 8 17 27 13 22 10 23 14 1 15 22
4 2 10 26 24
12 12 13 26 9 6 1 5 25 17 10 9 3
3 9 10 6
3 15 27 28
1 2
9 14 23 12 15 2 7 8 3 18
9 21 19 18 13 15 4 29 25 12
14 13 12 3 10 9 17 28 28 16 29 18 29 21 30
3 14 6 4
3 16 16 7
81 15
1 10
8 10 26 1 3 9 13 5 18
3 14 16 10
9 14 28 19 27 27 15 10 24 10
2 1 18
2 18 12
3 20 2 28
5 2 30 15 6 26
3 10 9 10
10 29 15 9 9 11 27 15 13 12 25
14 5 4 8 22 7 4 3 8 1 18 2 22 2 29
9 26 9 18 27 20 8 12 20 17
14 16 23 26 19 9 18 23 12 17 7 19 12 1 18
4 10 19 25 4
10 11 29 18 20 26 7 28 7 26 6
65 13
14 8 13 29 17 28 10 3 4 20 14 26 8 17 6
10 3 9 5 13 7 14 25 24 20 22
12 7 27 4 1 16 3 14 14 11 11 16 13
12 5 18 1 12 26 6 30 28 14 26 2 13
4 26 28 21 9
4 28 5 15 28
9 17 3 26 27 13 3 1 18 8
6 11 19 5 16 10 24
12 6 25 26 7 13 23 19 21 4 16 18 18
12 22 26 8 9 22 21 12 14 30 19 23 10
7 19 18 1 4 9 28 21
6 4 3 18 22 15 22
10 24 1 27 7 27 4 16 10 16 27
60 18
11 8 9 27 19 26 27 22 26 24 12 21
9 6 1 11 13 14 2 28 14 28
9 10 23 20 12 1 8 27 8 6
6 8 2 6 4 29 28
13 22 1 12 11 29 4 13 11 17 6 8 30 3
12 2 17 23 5 17 22 1 16 27 26 24 29
12 27 19 21 9 2 13 20 4 11 15 16 13
3 21 13 23
11 16 16 2 1 20 10 22 20 26 11 7
3 9 30 7
7 12 15 20 25 27 24 5
7 1 10 4 14 22 18 28
1 3
13 29 14 1 13 3 26 23 1 6 23 23 4 11
6 18 22 28 6 7 25
3 8 4 4
1 17
7 10 15 23 23 14 6 23
85 10
12 10 22 7 2 14 10 4 18 19 25 7 25
12 1 25 1 26 28 21 4 18 22 19 2 14
10 29 28 11 20 29 12 11 5 13 17
14 8 4 25 2 2 19 25 3 13 25 20 2 15 16
3 29 26 12
6 11 10 2 13 29 30
6 1 27 28 17 2 5
8 26 28 14 6 23 8 10 17
11 11 24 4 29 22 29 10 25 9 11 18
3 1 10 6
77 16
11 10 7 7 22 25 5 5 30 27 4 10
5 22 20 28 26 11
7 16 12 6 17 23 15 7
7 24 5 16 21 7 26 20
6 17 14 2 13 13 20
12 22 24 29 4 22 16 14 2 2 25 29 18
3 6 16 2
7 12 17 12 19 4 23 16
12 6 17 24 10 6 9 24 21 30 27 12 15
10 6 8 26 26 17 5 1 25 28 14
2 6 17
2 10 9
5 21 6 17 15 15
2 15 30
9 14 18 16 29 19 13 28 15 9
9 11 1 9 30 14 14 5 30 30
73 10
6 27 5 17 3 12 30
11 11 4 1 29 12 21 9 24 18 15 24
4 25 25 2 16
6 8 13 30 7 18 29
14 6 26 6 8 29 6 25 9 9 17 29 20 29 30
8 8 14 29 2 1 15 3 16
11 2 28 6 8 8 26 28 5 21 3 13
13 8 29 20 9 15 10 20 5 9 17 12 23 16
11 15 30 8 30 6 9 20 11 9 19 7
4 23 19 30 5
51 8
13 19 8 1 20 19 5 21 6 17 5 13 21 19
4 28 10 10 29
9 20 29 9 18 26 2 7 25 28
6 24 23 16 1 23 27
2 28 17
5 6 13 29 26 1
6 16 3 11 14 23 23
6 2 10 29 3 16 23
67 13
7 22 20 9 15 17 20 4
3 28 9 30
4 26 22 20 11
10 22 16 17 14 19 18 24 17 20 1
4 11 6 18 25
11 26 1 25 16 4 20 13 4 11 1 29
6 12 10 26 3 17 12
1 6
1 2
10 10 24 15 12 21 2 6 8 20 6
2 27 1
9 9 4 23 1 3 25 12 4 12
1 20
62 3
13 5 16 1 6 1 7 17 22 9 23 29 20 20
12 16 21 5 16 16 19 17 18 13 20 13 16
14 3 30 18 11 4 25 3 9 26 10 26 17 10 10
99 9
13 28 14 11 2 21 27 13 7 6 17 19 19 3
8 13 2 24 23 6 18 26 14
1 27
11 21 6 10 27 26 1 17 1 11 10 21
1 22
4 5 9 8 23
13 13 27 12 6 11 9 16 28 23 20 24 23 3
1 3
1 8
43 7
14 5 17 21 4 8 2 30 8 1 14 18 13 10 22
11 20 30 26 18 14 15 3 29 9 24 1
12 1 25 6 1 21 22 13 16 22 14 15 29
12 28 9 18 29 22 20 19 13 15 28 27 22
12 25 30 15 17 29 8 3 27 30 23 18 12
1 1
9 15 22 22 5 22 1 3 13 21
53 17
7 2 5 10 2 29 2 8
3 22 15 1
10 14 16 6 17 16 28 3 30 19 16
8 2 9 29 14 21 3 23 17
11 27 19 28 18 20 5 24 11 12 17 29
7 2 26 4 17 23 28 8
8 14 4 5 14 24 11 4 26
5 12 30 29 30 27
8 11 2 2 14 5 18 4 21
3 29 16 27
13 14 26 8 19 29 13 2 22 15 27 18 17 8
9 7 30 6 15 10 29 8 15 3
9 18 15 27 8 1 23 21 14 10
4 2 9 2 25
8 8 21 1 16 28 10 23 19
11 29 21 5 6 5 7 22 15 21 18 22
7 2 12 26 11 2 19 11
89 13
5 25 3 25 10 23
10 2 3 10 22 23 14 27 20 12 18
2 3 27
5 15 28 29 3 9
10 21 19 8 4 13 2 28 8 12 20
5 5 23 13 27 7
14 15 26 8 25 29 2 21 24 8 19 14 10 19 5
1 29
4 26 12 14 23
11 17 5 21 21 27 25 9 25 13 24 21
6 18 19 13 8 4 12
14 17 22 6 21 14 5 25 9 16 30 1 26 16 5
14 6 23 3 15 10 15 8 30 26 25 10 30 24 6
75 12
12 24 17 5 29 13 21 7 20 20 30 15 5
14 15 10 11 17 16 20 23 15 19 10 1 20 1 25
1 11
4 9 27 14 5
5 27 25 24 8 14
7 23 10 18 7 11 28 15
11 17 29 3 27 30 4 17 30 28 3 2
11 11 28 30 7 15 18 1 8 26 6 30
10 15 9 16 18 7 22 14 15 12 8
6 11 11 20 2 30 22
11 24 25 1 16 1 7 3 24 14 28 29
11 7 6 13 14 23 19 27 28 26 30 5
56 2
10 18 3 7 1 6 23 17 28 8 10
2 10 3
100 4
3 8 6 4
2 5 10
4 1 3 3 7
4 50 14 23 8
20 3
3 4 6 8
2 5 10
4 1 3 5 5
5 3
3 6 4 8
2 10 6
4 7 3 1 7
