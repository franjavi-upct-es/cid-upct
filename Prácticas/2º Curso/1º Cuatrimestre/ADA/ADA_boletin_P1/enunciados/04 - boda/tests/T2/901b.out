no solution
no solution
98
no solution
74
99
no solution
no solution
72
16
79
no solution
61
no solution
45
no solution
no solution
71
84
no solution
no solution
77
80
no solution
no solution
50
55
74
no solution
98
90
no solution
28
81
no solution
no solution
65
no solution
85
no solution
73
51
67
62
99
43
no solution
89
75
38
75
19
no solution
